js/index.js

```diff
@@ -40,3 +40,106 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// This is the circle img in center
+const slogan = document.getElementById("cta-img");
+slogan.setAttribute('src', siteContent["cta"]["img-src"]);
+
+// Nav titles is here
+const links = document.querySelectorAll("nav > a");
```

Comment:
Nice querySelectorAll with "nav > a" selecting all the anchor tags. "nav a" would have worked as well